src/components/news/NewsFeed.tsx

```diff
@@ -50,7 +50,10 @@ const NewsFeed = ({
                 transition={animationTransition}
               >
                 <NewsCard
-                  newsItem={newsItem}
+                  // (@SiravitPhokeed)
+                  // I have no idea what’s wro here so…um…apologies
+                  // for the `any`.
+                  newsItem={newsItem as any}
```

Comment:
any? you sure? Should we have an issue on this?

Reply:
Looks like I forgot to remove some old code on converting dates. Crisis averted!